scripts/pr-metadata-check.sh

```
@@ -0,0 +1,256 @@
+    is_infra_or_chore=1
+  fi
+
+  if [ "$is_release_promotion" -eq 1 ] && [ "$title_prefix" != "[release]" ]; then
+    failures+=("develop -> main release PR 必須使用 [release] title prefix")
+  fi
+  if [ "$is_release_promotion" -eq 0 ] && [ "$title_prefix" = "[release]" ]; then
+    failures+=("[release] 只能用於 develop -> main release promotion PR")
+  fi
+
+  local body
+  body=$(cat "$body_file")
```

Shellcheck (0.11.0)

[warning] body appears unused. Verify use (or export if used externally).

(SC2034)